src/components/Counter/Counter.spec.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
```

Comment:
I see that you removed the testkit tests, but they should be part of this test.

Reply:
done